src/Sulu/Bundle/SecurityBundle/Resources/js/index.js

```diff
@@ -1,10 +1,11 @@
 // @flow
 import {bundleReady, initializer} from 'sulu-admin-bundle/services';
 import {fieldRegistry} from 'sulu-admin-bundle/containers';
-import Permissions from './containers/Form/fields/Permissions';
+import {Permissions, RoleAssignments} from './containers/Form';
```

Comment:
Same as in previous PR, imports within the same bundle should not use that notation.

Reply:
It's in the index.js file, therefore not important here.